src/index.js

```diff
@@ -1 +1,19 @@
-'use strict';

const scale = require('./mode');
const chord = require('./chord');
const clip = require('./clip');
const pattern = require('./pattern');
const midi = require('./midi');
const scales = require('./modes');

let modes = Object.keys(scales);

// Allow scale to be denoted by mode as well
module.exports = {mode: scale, scale, chord: chord.getChord, listChords: chord.listChords, modes, scales: modes, clip, pattern, midi};
\ No newline at end of file
+'use strict';
+const scale = require('./mode');
+const chord = require('./chord');
+const clip = require('./clip');
+const pattern = require('./pattern');
+const midi = require('./midi');
+const scales = require('./modes');
+function setMiddleC(octaveIndex){
+    if(typeof octaveIndex !== "number" || Math.floor(octaveIndex) !== octaveIndex){
```

Comment:
Once you move the validation to the function definition to midi.js, you could replace the `if` block with,
```
assert(Number.isInteger(octaveIndex), 'Octave index should be an integer');
```